doc/COMPILE.md

```diff
@@ -606,6 +606,27 @@ running cmake like to enable debug symbol.
 For Gentoo it's recommended to emerge sys-apps/lsb-release to name the package
 right even thou not required.
 
+### With Flatpak (Beta)
+
+It is also possible to build a Flatpak package containing Elektra:
+
+```bash
+# install flatpak and flatpak-builder, using the package manager of your distro
+$ apt install flatpak flatpak-builder
+
+# download the runtime and sdk
+$ flatpak install flathub org.kde.Platform//5.15 org.kde.Sdk//5.15
+
```

Comment:
cd /path/to/elektra/source missing for me here.

Reply:
Thanks for the comment! I have added it to the docs.